modin/pandas/base.py

```diff
@@ -3420,13 +3533,27 @@ def __init__(
             base,
             on,
             level,
+            origin,
+            offset,
         ]
         self.__groups = self.__get_groups(*self.resample_args)
 
     def __get_groups(
-        self, rule, axis, closed, label, convention, kind, loffset, base, on, level
+        self,
+        rule,
+        axis,
+        closed,
+        label,
+        convention,
+        kind,
+        loffset,
+        base,
+        on,
+        level,
+        origin,
+        offset,
     ):
-        if axis == 0 or axis == "index":
```

Comment:
Why `index` value is not acceptable now?

Reply:
Because after call `_get_axis_number`, we will have either `1` or `0`.